metron-platform/metron-parsers/README.md

```diff
@@ -337,6 +337,28 @@ The following config will rename the fields `old_field` and `different_old_field
                       ]
 }
 ```
+* `REGEX_ROUTING` : This transformation lets users set an output field to one of a set of possibilities based on matching regexes. This has functional overlap with Stellar match statements, but the syntax is more bearable for large sets of conditionals.
```

Comment:
I don't think we need to editorialize so much about the match statements here.
"This transformation is useful when the number or conditions are large enough to make a stellar language match statement unwieldy" would be better.

Reply:
Yeah, that was wrong of me.  I had a bad night, when I wrote that.  My bad.  Will correct right away.  `match` is and continues to be awesome. :)